src/sentry/static/sentry/app/views/stream/index.jsx

```diff
@@ -12,12 +17,50 @@ const StreamContainer = createReactClass({
     setProjectNavSection: PropTypes.func,
   },
 
+  mixins: [ProjectState, Reflux.listenTo(StreamTagStore, 'onStreamTagChange')],
```

Comment:
Would it make sense to move tags into a HoC?

Reply:
Perhaps in future, not now since tags need to be handled differently in the stream than everywhere (exclude "environment")

Reply:
(We don't need to add the HoC for this PR)

HoC responsibility would be to just supply tags, stream can filter like it's doing with `filterTags`

Reply:
gotcha